pkgs/by-name/gh/gh-eco/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+  fetchpatch,
+}:
+let
+  version = "0.1.3";
+in
+buildGoModule {
+  pname = "gh-eco";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "coloradocolby";
```

Comment:
The repo is now https://github.com/jrnxf/gh-eco

Do you think you can update this in this PR too ?

Reply:
More then happy too.